docs/index.md

```diff
@@ -9,3 +9,13 @@ Here’s how you can get started with the Coiote DM – Azure IoT integration:
 [Configure integration with Azure IoT Central](Azure_IoT_Central_integration/Configuring_Azure_IoT_Central_integration_extension.md){: .md-button .md-button--big }
 
 [Configure integration with Azure IoT Hub](Azure_IoT_Hub_integration/Configuring_Azure_IoT_Hub_integration_extension.md){: .md-button .md-button--big }
+
+## Tutorial: Coiote IoT Device Management & Azure IoT Hub integration
```

Comment:
I think this is not the best place for that video. This should be at the beginning of Air quality monitoring page because this page is a "transcription" of that video. In my opinion, there should be a short video with an IoT Integration overview - max 2 minutes or a simple diagram which shows that coiote allows to connect LPWAN devices to Azure, and that's all.

Reply:
agreed